<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FRegistrarempleado</class>
 <widget class="QFrame" name="FRegistrarempleado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datos del empleado</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_nombre">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Apellidos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="le_apellido1">
          <property name="maximumSize">
           <size>
            <width>600000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_apellido2">
          <property name="maximumSize">
           <size>
            <width>600000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_nif">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>NIF</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_nif">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMask">
         <string>&gt;a99999999-A; </string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Dirección</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tipo de vía</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="cb_tipovia">
        <item>
         <property name="text">
          <string>Calle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Avenida</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paseo</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Nombre de vía</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_via">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Portal</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="le_portal">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Escalera</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="le_esc">
        <item>
         <property name="text">
          <string>Izquierda</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Derecha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Centro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Piso</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="le_piso">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Puerta</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="le_puerta">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Teléfono</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLineEdit" name="le_telefono">
        <property name="maximumSize">
         <size>
          <width>600000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Puesto de trabajo</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QComboBox" name="cb_puesto">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Piloto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Copiloto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Administrativo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jefe de departamento</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jefe de sección</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mecánico</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Personal limpieza</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btn_cancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_registrar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Registrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
